daemon/util/testnet_seeds/maker_seed

```diff
@@ -0,0 +1 @@
+%�_�U���]�v@cz���S�RJ�����o��o{���T�&h���Ƭu_�֬�h*(��BBm^�[6�Nm�I�+댵[cm���G�j�uC���>�x�A��L�Ccx��ZE��X#�z��}��ڀ�lA*�}"ڬ�I�p�`��lr=B�ٳ��T7_)D�p�,k�T�R�r��T��A�$����Ψ���^���VSA�+ѓ���^F�\=��U^H{�K�/ ,��è(��E�#8KO��ɨ��׹
```

Comment:
At the moment, we can only specify data-directory so in order to use these, we need to point the daemons to this via `--data-dir=./util/testnet_seeds`. That will also dump the database into here.

You didn't extend the `.gitignore` which means the DB  will always show up as part of the diff yeah?
Can we just define two data-dirs and only allow the seeds to be committed to Git? The naming of the dirs currently suggests that it only stores the seeds which is not true.

Reply:
Also, can we extend the README to document how to start the daemons to use these seeds?

Reply:
I did **not** see those are ready to use in this dir! I expected everyone to copy them where needed. They are not supposed to change. I don't see a reason for optimising this at the moment. If you feel the need for this please create a follow up PR.